packages/batch-submitter/src/exec/run-batch-submitter.ts

```diff
@@ -15,19 +15,45 @@ import {
   TX_BATCH_SUBMITTER_LOG_TAG,
 } from '..'
 
+const environment = process.env.NODE_ENV
+const release = `batch-submitter@${process.env.npm_package_version}`
```

Comment:
This tag is tightly coupled to Sentry, so we may want to keep it nearer to that code?

Reply:
^should we tie metrics to releases too @snario ?

Reply:
Hm... we ought to reference the version of the software running in metrics we send over. Good point. Yeah.